atlas-hub/pre-conf/solr/currency.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" ?>
```

Comment:
SOLR configs files might be not be needed since we are using elastic search as indexing backend.
need to verify if there no strict dependency on it and remove if not needed.